app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -16,7 +16,7 @@ def submit
     session[:reference_id] = submission_reference
 
     json_validation = validate_against_form_response_schema(sanitised_session)
-    if json_validation.fail
+    if json_validation.invalid?
```

Comment:
```suggestion
    unless json_validation.valid?
```
Generally `valid?` is called to test if something is ok. It would be nice if we could repeat that pattern here.

Can this change in name be fixed up to the previous commit to make it easier to traverse the history.

Reply:
I've tended to find people don't like `unless` so used a form that matched `if` cleanly. Happy to go for `unless` pattern.